docs/DeveloperGuide.adoc

```diff
@@ -1512,16 +1583,61 @@ Expected: No employee at index 0. Error details is shown in the status message.
 
 ==== Leave application
 
-==== List leave application
+. Requesting leave application
+
+.. Prerequisites: User must be logged in.
+.. Test case: `leave date/01/01/2019` +
+   Expected: Leave application requested. Details of the leave application will be reflected in the leave panel.
+.. Test case: `leave date/01/02/2018` +
+   Expected: No leave application created. Error details shown in the status message.
 
 ==== Filter leave application
 
+. Filter leave application
+
+.. Test case: `filterleave S8570520Q` +
+   Expected: List all leave application under the user with NRIC,S8570520Q.
+.. Test case: `filterleave S1212121A` , `filterleave S1313131A` +
+   Expected: 0 leaves listed. This is due to no such leave application under S1212121A and S1313131A.
+.. If NRIC is not valid such as: 'filterleave S8570520', 'filterleave 8570520Q', 'filterleave 8570520' +
```

Comment:
Suggestion: can use `filterleave S5570520`, ....... instead of 'filterleave S8570520', ...... But I'm fine with that.